Facade_Engine/Query/PrimaryPropertyName.cs

```diff
@@ -82,6 +82,9 @@ public static string PrimaryPropertyName(this FrameEdge frameEdge)
         [Output("propName", "The name of the element's construction property")]
         public static string PrimaryPropertyName(this IEdge edge)
         {
+            if (edge == null)
```

Comment:
Same as above.

Reply:
This was set up as was, so figured the discipline were happy with an empty string with no warning. If warnings are desired, I am leaving that to the discipline to add either to this PR before merge, or after in their own PR :smile: